src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto2";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    required bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor.
+    required float break_beam_reading = 2;
+}
+
+/* Data about the status of the chipper and kicker */
+message ChipperKickerStatus
+{
+    /* Frankie Robot Fields */
+    // Indicates the time elapsed since chipper was last fired in ms
+    // UINT64_MAX indicates chipper was never fired
+    optional uint64 time_since_chipper_fired = 2;
```

Comment:
we should adjust these to to be 1,2,3

technically all field numbers until 15 we are safe, but might as well keep it in order

Reply:
done